docs/players/Installation_Android.md

```diff
@@ -11,13 +11,29 @@ Installation is a two step process, at first you need to install game, then you
 
 ## Step 2: Installing Heroes III data files
 
-- Install Heroes III on your PC or extract Heroes III data files from gog installer
-- Connect your device to PC and enable file transfer.
-- Copy "Data", "Maps" and "Mp3" from Heroes III to any folder on  mobile device, then open VCMI, select Heroes III data import option, then select the folder where you copied Heroes III data.
+a) If you purchased HoMM3 on [GOG](https://www.gog.com/de/game/heroes_of_might_and_magic_3_complete_edition), you can download the required files directly on your device.
 
 gog.com download page:
 ![GoG-Installer](images/gog_offline_installer.png)
```

Comment:
In theory we could also post direct download links here (also for chronicles all-in-one), to save players some time.

Download links from gog.com should be permanent, same for all users, and obviously only work if user own h3 & logged into gog.com. So should be safe to show them here